R/shared-methods.R

```diff
@@ -149,6 +151,10 @@ DataClass <- R6::R6Class(
     #' @field target_regions A character vector of regions to filter for. Used
     #' by the `filter method`.
     target_regions = NULL,
+    #' @field process_options A named list of string function names .giving a
```

Comment:
I think this would be better as a list of functions rather than a list of character strings referring to functions.

Reply:
if making that change perhaps a good idea to call it `process_fn`